src/imlib.c

```diff
@@ -35,6 +35,7 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <netdb.h>
+#include <unistd.h>
```

Comment:
On Linux, mkstemp, mkdtemp, and mkstemps are defined in `stdlib.h` (which is also missing from imlib.c, but both stdlib.h and unistd.h are included in feh.h). Is that different an MacOS?

Reply:
> Is that different an MacOS?

Yes! I see from [a Linux manpage](https://linux.die.net/man/3/mkstemp) that on Linux all three are indeed in stdlib.h. [opengroup agrees](https://pubs.opengroup.org/onlinepubs/009604499/functions/mkstemp.html) that mkstemp and mkdtemp are in stdlib.h; I couldn't find mkstemps at opengroup. Per [a macOS manpage](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/mkstemp.3.html), on macOS all three are in unistd.h. The manpage says "The include file <unistd.h> is necessary and sufficient for all functions." Here is [Apple's unistd.h](https://opensource.apple.com/source/Libc/Libc-1439.40.11/include/unistd.h.auto.html) showing them declared there. Per [a FreeBSD manpage](https://www.freebsd.org/cgi/man.cgi?query=mkstemp&sektion=3&format=html), mkdtemp and mkstemp are in stdlib.h while mkstemps is in unistd.h. So things are a bit less standardized than I would have hoped.

> both stdlib.h and unistd.h are included in feh.h

Indeed I neglected to notice this. So I should remove the second commit from this PR.

Reply:
Yeah. I've taken the liberty of simply cherry-picking the first commit into feh's main branch. Thank you for the patch!